spec/features/movies/show_spec.rb

```diff
@@ -1,31 +1,101 @@
 require 'rails_helper'
 
 RSpec.describe 'Movies detail page' do
-  describe 'As an authenticated user, when I visit a movies detail page, I see' do
+  describe 'As an authenticated user, when I visit a movies detail page' do
     before :each do
       @user = FactoryBot.create(:user)
       allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(@user)
     end
 
-    it "I can see a movie's title" do
+    it 'I see a button to create a viewing party' do
       VCR.use_cassette('show_movie_details') do
         visit(movie_show_path('524'))
-        expect(page).to have_content('Casino')
+        expect(page).to have_button('Create Viewing Party')
       end
     end
 
-    it 'I see a button to create a viewing party' do
+    xit 'I click create a viewing party, I am taken to the new event page' do
       VCR.use_cassette('show_movie_details') do
+        # this test will need to be reactiviated once we have the party page working
         visit(movie_show_path('524'))
-        expect(page).to have_button('Create Viewing Party')
+        click_button('Create Viewing Party')
+        expect(current_path).to eq(party_create_path)
       end
     end
 
-    it 'When I click create a viewing party, I am taken to the new event page' do
+    it 'I see the movies title' do
       VCR.use_cassette('show_movie_details') do
         visit(movie_show_path('524'))
-        click_button('Create Viewing Party')
-        expect(current_path).to eq(party_create_path)
+        within '#title' do
+          expect(page).to have_content('Casino')
+        end
+      end
+    end
+
+# should these be dynamic here? @movie[:vote_average]
```

Comment:
@dhalverson Good q. I am also wondering how we can make these test instances dynamic.